src/xgov-dapp/src/features/vote-creation/RoundInfo.tsx

```diff
@@ -24,7 +25,7 @@ const formSchema = zfd.formData({
   start: zfd.text(),
   end: zfd.text(),
   communityGrantAllocation: zfd.numeric(z.number().positive().min(1, 'Must be at least 1')),
-  proposalFile: zfd.text(z.string().trim().min(1, 'Required').superRefine(validateProposalCsv)),
+  proposalFile: zfd.text(z.string().trim().min(1, 'Required').transform(validateProposalCsv)),
```

Comment:
What about changing `validateProposalCsv` to `validateAndSantizeProposalCsv`?

Reply:
Absolutely!